proto/regen/ecocredit/basket/v1/tx.proto

```diff
@@ -12,14 +12,15 @@ option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket/ty
 // Msg is the regen.ecocredit.basket.v1 Msg service.
 service Msg {
 
-  // Create creates a bank denom which wraps credits.
+  // Create creates a basket.
```

Comment:
not sure if these should be updated to be more verbose like the `base` tx docs are? i kinda like the simplicity, letting the field docs be more specific.

thoughts @ryanchristo @aleem1314 ? or should we update with more description like `base`?

Reply:
I think it would be nice to provide more information. At least some explanation of what a basket is.

Reply:
ok will draft this and update later today

Reply:
fleshed out here https://github.com/regen-network/regen-ledger/pull/1559/commits/41e04202706f6a454502b6cc536ecfd8ad41db07